cmd/cmd.go

```diff
@@ -348,6 +360,13 @@ func (o *options) generateTemplate() (*template.Template, error) {
 			}
 			return string(b), nil
 		},
+		"tryParseJSON": func(text string) map[string]interface{} {
```

Comment:
Could you add the new functions to the "templates" section in README.md?
https://github.com/stern/stern#templates

Reply:
Definitely, thank you for pointing it out 🙇